lib/dirac_domain_wall_4d.cpp

```diff
@@ -178,7 +179,7 @@ namespace quda {
     }
 
     // we desire solution to full system
-    auto tmp = getFieldTmp(b[0].Even());
+    auto tmp = getFieldTmp(x[0].Even());
     for (auto i = 0u; i < b.size(); i++) {
```

Comment:
Is this being intentionally left as a loop over each separate 5-d spinor? All good if so, just wanted to make sure this wasn't an accidental oversight.

Reply:
This one is an oversight, thanks for the catch.

Aside: I have the `prepare` and `reconstruct` Dirac member functions as loops intentionally, rather than "vectorize" them, since we don't have multi-RHS solvers in place which would be needed for testing that functionality.  Once we do have those, I'll vectorize all those functions.

Reply:
GitHub formatting confused me for a minute there: confirming this one was intentional for the reason stated above: I'll update `prepare` / `reconstruct` in a subsequent PR.